### DISCUSSION TITLE
Error; limit exceeded......you should lower connections....?

### DISCUSSION THREAD

**Discussion by Cren0** [TumblThreeApp/TumblThree#341] [Q&A (Question & Answer)]
All of a sudden my program has experienced error message on all tumblr sites, and does not download anymore.

I get error on all blogs now, without making any changes to program....also tried to reinstall it and still nothing.
Error: limit exceeded, you should lower connections to tumblr api in settings...Now what should I do, if I have not changed anything at all...?

I have now tried to empty all logs, blogs, ja folders....and nothing.....
Any inside for this ?

**Comment by thomas694** [comment]
Normally the message means that you've downloaded too much. Also you shouldn't use Tumblr in multiple TumblThree/browser instances at the same time.
To reset it, you have to wait a couple of hours or a day. But you can also try to restart your "router" to get a new IP address.
